### PR TITLE
Camera Zoom, Camera Move.

### PR DESCRIPTION
**Pull request by lapalm**
Is very good.

### CODE REVIEW

File: Motoriis/GameManager.cpp [hunk 1]
```diff
@@ -135,8 +202,9 @@ void GameManager::mainLoop() {
 
 void GameManager::render() {
 
-	sf::RectangleShape shape(sf::Vector2f(3, 3));
```
Comment (lapalm) [inline Motoriis/GameManager.cpp]: Why you do this? D:

File: Motoriis/BforBuilding.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
```
Comment (JohnHonkanen) [inline Motoriis/BforBuilding.h]: What is this doing?

**Comment by lapalm** [comment]
Accept merge na0.

File: Motoriis/GameManager.cpp (hunk 1 above)
Comment (JohnHonkanen) [inline Motoriis/GameManager.cpp]: No you

File: Motoriis/BforBuilding.h (hunk 2 above)
Comment (lapalm) [inline Motoriis/BforBuilding.h]: Is very good yes?

File: Motoriis/GameManager.cpp (hunk 1 above)
Comment (lapalm) [inline Motoriis/GameManager.cpp]: You know why.

**Comment by JohnHonkanen** [comment]
Fix BforBuilding please. Doesnt do anything

**Pull request closed without merging**